### DISCUSSION TITLE
Getting more information from swagger

### DISCUSSION THREAD

**Discussion by vlombardi22** [cmu-sei/GHOSTS#164] [Q&A] [answered]
We are only getting around 50 entries from each machine while the grafana database and client log files have many more executions. Is there any way to increase the number of entries we get from swagger?

**Comment by sei-dupdyke** [comment, marked as answer]
Argh, this is a bug, I just worked a fix, and published an updated [docker container 6.2.5](https://hub.docker.com/layers/dustinupdyke/ghosts/6.2.5). This adds two parameters to the history methods: Skip and Take so that you can paginate however you wish. Sorry about that, can't believe we didn't find this one a lot sooner.
